packages/parameters/tests/e2e/secretsProvider.class.test.functionCode.ts

```diff
@@ -0,0 +1,57 @@
+import { Context } from 'aws-lambda';
+import { SecretsProvider } from '../../lib/secrets';
+import { TinyLogger } from '../helpers/tinyLogger';
+import { transform } from 'esbuild';
```

Comment:
Why do we import this and use it in the conditional? Not challenging, just asking because I'm not clear on it

Reply:
This does not make any sense 🤔 , it should be `options` in the conditional. Will change that.